client/src/components/HistoryViewer/HistoryViewer.scss

```diff
@@ -151,3 +151,44 @@
   }
 }
 
+.griddle-footer {
```

Comment:
I'm a bit concerned about the scope of these styles. Maybe we can limit this by wrapping all of the new styles within `.history-viewer`?